networking/private-cloud-function-from-onprem/main.tf

```diff
@@ -19,7 +19,7 @@
 ###############################################################################
 
 module "project-onprem" {
-  source          = "github.com/terraform-google-modules/cloud-foundation-fabric//modules/project?ref=v5.0.0"
+  source          = "../../modules/project"
   billing_account = var.billing_account_id
   name            = var.onprem_project_id
```

Comment:
I would use a single `projects_ids` variable, with `gcp` and `onprem` attributes.